app/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "tidepool-uploader",
   "productName": "tidepool-uploader",
-  "version": "2.35.0",
+  "version": "2.36.1",
```

Comment:
Shouldn't a hotfix only bump the patch version, and not the minor?

Reply:
The minor version (https://github.com/tidepool-org/uploader/pull/1380) hasn't been merged yet, so this PR can supersede that one, unless you think we should merge that one first?

Reply:
Nope, that's fine, and makes sense.